### PR TITLE
Remove Travis links and add Jenkins links.

### LINKED ISSUE #517: Travis CI deprecated?
The links to the travis CI (for LXD and LXC) don't seem to exist anymore.

I guess they should be replaced by Jenkins links.

### PR DESCRIPTION
**Pull request by toby63**
Closes #517.

Todo:
- [x] Add Github Actions links
- [x] Change other Structure.json files (language files) (only japanese version)

### CODE REVIEW

File: content/STRUCTURE.json [hunk 1]
```diff
@@ -340,10 +340,10 @@
      "generator": "link",
      "meta": {"url": "https://github.com/lxc/distrobuilder"}},
 
-    {"path": "/distrobuilder/travis-ci/",
-     "menu": ["distrobuilder", "Travis CI"],
+    {"path": "/distrobuilder/jenkins-ci/",
+     "menu": ["distrobuilder", "Jenkins CI"],
      "generator": "link",
-     "meta": {"url": "https://travis-ci.org/lxc/distrobuilder"}},
+     "meta": {"url": "https://jenkins.linuxcontainers.org/view/Distrobuilder/"}},
```
Comment (stgraber) [inline content/STRUCTURE.json:346]: https://github.com/lxc/distrobuilder/actions

File: content/STRUCTURE.json [hunk 2]
```diff
@@ -280,6 +275,11 @@
      "generator": "link",
      "meta": {"url": "https://github.com/lxc/lxcfs"}},
 
+    {"path": "/lxcfs/jenkins-ci/",
+     "menu": ["LXCFS", "Jenkins CI"],
+     "generator": "link",
+     "meta": {"url": "https://jenkins.linuxcontainers.org/view/LXCFS/"}},
```
Comment (stgraber) [inline content/STRUCTURE.json:281]: https://github.com/lxc/lxcfs/actions

File: content/STRUCTURE.json [hunk 3]
```diff
@@ -107,10 +107,10 @@
      "generator": "link",
      "meta": {"url": "https://github.com/lxc/lxd"}},
 
-    {"path": "/lxd/travis-ci/",
-     "menu": ["LXD", "Travis CI"],
+    {"path": "/lxd/jenkins-ci/",
+     "menu": ["LXD", "Jenkins CI"],
      "generator": "link",
-     "meta": {"url": "https://travis-ci.org/lxc/lxd"}},
+     "meta": {"url": "https://jenkins.linuxcontainers.org/view/LXD/"}},
```
Comment (stgraber) [inline content/STRUCTURE.json:113]: https://github.com/lxc/lxd/actions

File: content/STRUCTURE.json [hunk 4]
```diff
@@ -206,11 +206,6 @@
      "generator": "link",
      "meta": {"url": "https://jenkins.linuxcontainers.org/view/LXC/"}},
 
-    {"path": "/lxc/travis-ci/",
-     "menu": ["LXC", "Travis CI"],
-     "generator": "link",
-     "meta": {"url": "https://travis-ci.org/lxc/lxc"}},
```
Comment (stgraber) [inline content/STRUCTURE.json:212]: https://github.com/lxc/lxc/actions

**Comment by stgraber** [comment]
As I said in the related issue, we switched those to Github Actions, so we should just update the links to the matching Github Actions URL.

For cgmanager, it's a dead project so that one is fine to just drop as you did.

**Comment by toby63** [comment]
> As I said in the related issue, we switched those to Github Actions, so we should just update the links to the matching Github Actions URL.

As I say [above](https://github.com/lxc/linuxcontainers.org/pull/521#issue-728164317), the problem with github actions seems to be that downloads and logfiles are only available with a github account.
I don't think that is really useful to put that on the website, but if you want that anyway, I can of course add that.

In that case:
Should I then add both Github actions and Jenkins?
Or just Github Actions?

**Comment by stgraber** [comment]
The test results are available and that matches what we had on Travis. Travis never built artifacts for us.
We do use Github Actions for some extra stuff like the LXD artifacts but not having those be available to non-registered users shouldn't be a blocker here (as annoying as it is).

**Comment by toby63** [comment]
Ok.

File: content/STRUCTURE.json [hunk 5]
```diff
@@ -270,6 +285,16 @@
...
 
+    {"path": "/lxcfs/github-actions/",
+     "menu": ["LXCFS", "GitHub Actions"],
+     "generator": "link",
+     "meta": {"url": "https://github.com/lxc/lxcfs/actions"}},
+
+    {"path": "/lxcfs/jenkins-ci/",
+     "menu": ["LXCFS", "Jenkins CI"],
+     "generator": "link",
+     "meta": {"url": "https://jenkins.linuxcontainers.org/view/LXCFS/"}},
```
Comment (stgraber) [inline content/STRUCTURE.json:296]: This should be dropped, there is no meaningful CI tests done for LXCFS on Jenkins, it's just basic commit signature stuff.

File: content/STRUCTURE.json [hunk 6]
```diff
@@ -330,6 +355,16 @@
...
 
+    {"path": "/distrobuilder/github-actions/",
+     "menu": ["distrobuilder", "GitHub Actions"],
+     "generator": "link",
+     "meta": {"url": "https://github.com/lxc/distrobuilder/actions"}},
+
+    {"path": "/distrobuilder/jenkins-ci/",
+     "menu": ["distrobuilder", "Jenkins CI"],
+     "generator": "link",
+     "meta": {"url": "https://jenkins.linuxcontainers.org/view/Distrobuilder/"}},
```
Comment (stgraber) [inline content/STRUCTURE.json:366]: Same as LXCFS, there's no meaningful tests being run on Jenkins for distrobuilder.

**Comment by stgraber** [comment]
Thanks!

**Pull request merged**